x-pack/plugins/cases/public/components/all_cases/assignees_filter.tsx

```diff
@@ -91,44 +91,46 @@ const AssigneesFilterPopoverComponent: React.FC<AssigneesFilterPopoverProps> = (
   const isLoadingData = isLoading || isLoadingSuggest;
 
   return (
-    <UserProfilesPopover
-      isOpen={isPopoverOpen}
-      closePopover={togglePopover}
-      panelStyle={{
-        minWidth: 520,
-      }}
-      button={
-        <EuiFilterButton
-          data-test-subj="options-filter-popover-button-assignees"
-          iconType="arrowDown"
-          onClick={togglePopover}
-          isLoading={isLoadingData}
-          isSelected={isPopoverOpen}
-          hasActiveFilters={selectedAssignees.length > 0}
-          numActiveFilters={selectedAssignees.length}
-          aria-label={i18n.FILTER_ASSIGNEES_ARIA_LABEL}
-        >
-          {i18n.ASSIGNEES}
-        </EuiFilterButton>
-      }
-      selectableProps={{
-        onChange,
-        onSearchChange,
-        selectedStatusMessage,
-        options: searchResultProfiles,
-        selectedOptions: selectedAssignees,
-        isLoading: isLoadingData || isUserTyping,
-        height: 'full',
-        searchPlaceholder: i18n.SEARCH_USERS,
-        clearButtonLabel: i18n.CLEAR_FILTERS,
-        emptyMessage: <EmptyMessage />,
-        noMatchesMessage: !isUserTyping && !isLoadingData ? <NoMatches /> : <EmptyMessage />,
-        limit: MAX_ASSIGNEES_FILTER_LENGTH,
-        limitReachedMessage,
-        singleSelection: false,
-        nullOptionLabel: i18n.NO_ASSIGNEES,
-      }}
-    />
+    <EuiFilterGroup>
```

Comment:
I need to include this because, without it, the selectable styling breaks. It means that we are wrapping every single selectable in a filter group (a one filter only filter group) which might not be ideal semantically